### ISSUE TITLE
Qnap web ui crash

### ISSUE THREAD

**Issue by fdm91gh** [Yannik/qnap-letsencrypt#10]
**Is this the latest firmware for your device?**
Yes

**Issue Summary (provide relevant error messages and log output):**
Certificate signed.crt is empty and qnap web ui stops working (no answer from server) after executing renew_certificate.sh script. SSH, mobile apps and FTP keep working as they should.
No error reported during scripts execution

**Comment by fdm91gh** [comment]
EDIT:
i repeated the entire procedure right now and finally could get the signed.crt and all keys needed but unfortunately, the Qnap UI keeps crash. This time I was able to restore it without need to reset entire NAS simply restoring stock certificate from a webpage still open.
What could be the problem?

**Comment by Yannik** [comment]
Please provide the output of the renew_certificate.sh script.

**Comment by fdm91gh** [comment]
Hi Yannik and first of all thanks for your support, your script and all your work.
Anyway, here is my output, it seems there is an issue with Host Controller protocols

```
[/share/HDA_DATA/qnap-letsencrypt] # ./renew_certificate.sh
Checking whether to renew certificate on Tue, 10 May 2016 18:09:36 +0200
Renewing certificate...
Stopping Qthttpd hogging port 80..
Shutting down Qthttpd services: OK.
Started python SimpleHTTPServer with pid 7469
Serving HTTP on 0.0.0.0 port 80 ...
Parsing account key...
Parsing CSR...
Registering account...
Registered!
Verifying fdm91.myqnapcloud.com...
192.168.1.1 - - [10/May/2016 18:09:50] "GET /.well-known/acme-challenge/O8lWzMeI                                                                                                                                                             sAIRCc2KOz5qYG1z5mvuXb1yr9Phj4B_xS4 HTTP/1.1" 200 -
outbound1.letsencrypt.org - - [10/May/2016 18:09:53] "GET /.well-known/acme-chal                                                                                                                                                             lenge/O8lWzMeIsAIRCc2KOz5qYG1z5mvuXb1yr9Phj4B_xS4 HTTP/1.1" 200 -
fdm91.myqnapcloud.com verified!
Signing certificate...
Certificate signed!
Downloading intermediate certificate...
2016-05-10 18:09:59 URL:https://letsencrypt.org/certs/lets-encrypt-x3-cross-sign                                                                                                                                                             ed.pem [1647/1647] -> "-" [1]
Stopping stunnel and setting new stunnel certificates...
Shutting down apache proxy: OK
Done! Service startup and cleanup will follow now...
Start apache proxy: OK
df: /share/external/Etron xHCI Host Controller: Protocol error
df: /share/external/Etron xHCI Host Controller: Protocol error
Starting Qthttpd services: OK
```

I really don't have any ideas about it, sorry, and thanks in advance for your help :)

**Comment by Yannik** [comment]
That is interesting. The script is running absolutely fine. Unfortunately, you did not fill the issue template when you opened the issue, so I cannot even know which model/firmware you are experiencing this on.

**Comment by fdm91gh** [comment]
Sorry for that, I submitted the issue from phone and thinked that in this case the model-firmware was not so important, my fault :)

Anyway, I'm executing the script on a Qnap TS-212P running the latest version 4.2.0 (2016-03-11) as follow:

```
Platform = MARVELL_6281
Model = TS-212
Version = 4.2.0
Build Number = 20160311
Build Date = 2016-03-11
Model Name = TS-210
```

Unfortunately, as I told in the previous post, as soon as I apply the cert/RSA/intermediate cert I get a Qnap UI crash both remotely and locally. I tried repeatedly, but nothing changed.

No issues with a self-signed openssl generated cert or with the payed myqnapcloud certificate I'm using right now. I'm really stuck at this point.
